worker/client.go

```diff
@@ -219,15 +222,23 @@ func (c *Client) ObjectEntries(ctx context.Context, path, prefix string, offset,
 	return
 }
 
+type DownloadObjectOption func(http.Header)
```

Comment:
can we move these to the `api` package?

Reply:
I don't think it belongs there tbh. It's neither a request nor response type and has nothing to do with the actual API.
It's a type exclusively used by the client. So I'd be rather confused about having to import another package when using it.

Reply:
cfr cyclic imports (I see your point though it is confusing)